app/scripts/components/globalComponents/leftMenu.jsx

```diff
@@ -138,10 +138,13 @@ class LeftMenu extends React.Component {
     }
 }
 
+const toolbarHeight = '56px'
```

Comment:
Why are you setting this variable and then using it in your styles object? Why not just set `top: 56` and `height: 56`?

Reply:
Mostly Notes for myself
Push up fixes for this